drivers/at86rf215/include/at86rf215_registers.h

```diff
@@ -0,0 +1,2401 @@
+/*
+ * Copyright (C) 2019 ML!PA Consulting GmbH
+ *               2014 Atmel Corporation
```

Comment:
I got that from [openwsn-fw](https://github.com/openwsn-berkeley/openwsn-fw/blob/develop/bsp/chips/at86rf215/at86rf215.h).

Reply:
You can download the official one from https://www.microchip.com/mplab/avr-support/advanced-software-framework and it will be under xdk-asf-3.47.0/thirdparty/wireless/avr2025_mac/source/tal/at86rf215/inc.

Can you split what is vendor file and what is CMSIS style register definition?

I also see that there are defines available for all register address definition, this would be the most annoying change request, so I'll limit this to a comment, but they could probably be replaced by the defines, although you could probably write a macro do it easily in vim.

Reply:
I'm feeling silly now that I didn't find that driver earlier…